release/build_release.sh

```diff
@@ -199,4 +200,12 @@ function print_variables {
 	echo ""
 }
 
+function create_release_properties_file {
+	{
```

Comment:
Incorrect syntax.
How did you test it?

Reply:
Why is this bad syntax? It works. We should switch the '{' with '('?